README.md

```diff
@@ -109,6 +112,16 @@ $ yay -S github-cli
 
 Install a prebuilt binary from the [releases page][]
 
+## Setting an Editor
```

Comment:
I pulled this copy directly from the [docs](https://cli.github.com/manual/installation). The docs assume the user knows how to set environment variables, but it may be a good idea to eventually add a tutorial which walks you step-by-step through this. Then we can link that here for those who don't know how to set env vars.

Reply:
Thank you for suggesting, but I'm not sure if this belongs in the README.

In general, most in-depth documentation and configuration settings will be covered in the docs (either through `gh help` or on our docs site) and I don't believe most of that belongs in the README; otherwise we have to update documentation in two places whenever we make changes.

Furthermore, we are planning to add configuration options for GitHub CLI which would allow people to set their preferred text editor _without_ knowing how to set environment variables. This would also work for Windows.

I will keep this open so my coworkers can weigh in, but in my view this change doesn't substantially improve our current documentation and when we ship config options, it will become obsolete anyway.

Reply:
Fair point! I was only to help. If you look at the issue I linked, Billy and I had a conversation about this. Clearly people aren't reading the docs, or don't know they exist because this issue keeps coming up.

I'm not saying this is the best solution, or the only solution. Once you allow users to configure it via the CLI, I think it'll be solved. The purpose of this was to serve as a temporary workaround.

Feel free to close though if you and the team decide you're going to pursue a different solution!

Reply:
You're right that this issue keeps popping up and that, in general, moving information to the README does improve discoverability. In this case, though, I'm leaning to improving this through better implementation rather than documentation changes, especially because we already have docs for this.

We could, for instance, notice when the person doesn't have any editor explicitly configured and give them a hint on the command line that they could run a specific command to set their preferred editor. That way we have a good chance of solving their problem even if they won't read the manual.

That's just my thoughts, though. I'm interested in what the rest of my team has to say?

Reply:
Thanks for the PR!

I do agree with mislav; we've got some things in flight to mitigate the EDITOR confusion and for now think we can leave it out of the README.